Realm/RLMProperty.h

```diff
@@ -92,4 +97,17 @@ RLM_ASSUME_NONNULL_BEGIN
 
 @end
 
+
+/**
+ This class describes a specific property on a given class.
+ */
+@interface RLMPropertyDescriptor : NSObject
+
++ (instancetype)descriptorWithClass:(Class)objectClass propertyName:(NSString *)propertyName;
```

Comment:
Swift generates warnings since it doesn't consider properties to have selectors:
```
examples/ios/swift-2.2/Backlink/AppDelegate.swift:27:75: warning: no method declared with Objective-C selector 'dogs'
    let owners = LinkingObjects(fromType: Person.self, property: Selector("dogs"))
```